dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js

```diff
@@ -44,39 +44,6 @@ const extendDealWithFacilities = async (deal) => {
   return mappedDeal;
 };
 
-const queryDeals = async (query, start = 0, pagesize = 0) => {
```

Comment:
was this used only for reports? :|

Reply:
Honestly i'm not sure anymore 😂 I think some used this, others used "transactions".